routes/v1.js

```diff
@@ -1,19 +1,21 @@
 import express from 'express';
 import UserController from '../controllers/user';
-import { validateSignup, returnValidationErrors } from '../middleware/validation';
+import ArticleController from '../controllers/articles';
+import { validateSignup, returnValidationErrors, validateArticle } from '../middleware/validation';
+import isUserVerified from '../middleware/verifyUser';
+import Auth from '../middleware/auth';
+
+const { createArticle } = ArticleController;
 
 const apiRoutes = express.Router();
 
-apiRoutes.post(
-  '/user',
-  validateSignup,
-  returnValidationErrors,
-  UserController.registerUser,
-);
+apiRoutes.route('/user')
+  .post(validateSignup, returnValidationErrors, UserController.registerUser);
+
+apiRoutes.route('/verify/:verificationId')
+  .get(UserController.verifyAccount);
 
-apiRoutes.get(
-  '/verify/:verificationId',
-  UserController.verifyAccount,
-);
+apiRoutes.route('/articles')
+  .post(Auth.verifyUser, isUserVerified, validateArticle, returnValidationErrors, createArticle);
```

Comment:
@kleva-j there's a `validateArticle` middleware here. Wouldn't it be a good point to trim the article content, title, etc?

Reply:
Yes I actually took care of trimming the article using a hook on the article model.

Reply:
@kleva-j Yeah I'm aware. I'm saying it'll be better you do it in the middleware, than in the db hook